evalml/tests/pipeline_tests/test_component_graph.py

```diff
@@ -458,36 +458,43 @@ def test_component_graph_fit_features(
     mock_X_t = pd.DataFrame(np.ones(pd.DataFrame(X).shape))
     mock_fit_transform.return_value = mock_X_t
     mock_fit.return_value = Estimator
-    mock_predict.return_value = pd.Series(y)
+    mock_predict_proba.return_value = pd.Series(y)
```

Comment:
Should we make this a DataFrame instead to mimic and make sure that when we have an output with more than one column?

Reply:
(oops, pressed add single comment too quickly!)